### PR TITLE
Linkify administrative citations (Tax Court and EPA)

### PR DESCRIPTION
**Pull request by v-anne**
This PR follows up on #4293 and #4310. It adds two more options for linking to administrative dockets, namely the US Tax Court's docket tool, and some EPA cases whose administrative docket numbers on PACER are not Federal Register citations.

### CODE REVIEW

**Comment by mlissner** [comment]
Sorry, @v-anne, I think I missed this one, thinking it was still a work in progress. It helps if you flag me for review when things are ready, so I can get it into somebody's workflow. I'll assign to Eduardo now. 

Thank you!

**Comment by v-anne** [comment]
No worries, @mlissner. I wasn't sure how to request a review (I thought Github has a feature to do that without needing the reviewer to add it to their queue).

File: cl/lib/model_helpers.py [hunk 1]
```diff
@@ -536,6 +536,28 @@ def linkify_orig_docket_number(agency: str, og_docket_number: str) -> str:
             )
             return f"https://www.nlrb.gov/case/{formatted_number}"
 
+    # US Tax Court pattern
+    if "Tax" or "Internal Revenue" in agency:
```
Comment (ERosendo) [inline cl/lib/model_helpers.py:540]: I think the if statement's condition is always true. We can replace it with a different expression:

```suggestion
    if any(x in agency for x in ("Tax", "Internal Revenue")):
```

Comment (v-anne) [reply cl/lib/model_helpers.py]: Thanks @ERosendo, fixed.

**Review by ERosendo — APPROVED** [review]

**Comment by mlissner** [comment]
All set for auto-merge, thank you!

**Pull request merged**